### PR TITLE
Transform-Form-State function was created

### PR DESCRIPTION
**Pull request by yuliia-dronova**

### CODE REVIEW

**Review by Danil-Hodytskyi — APPROVED** [review]
Good Job

File: src/transformState.js [hunk 1]
```diff
@@ -5,7 +5,30 @@
...
+        }
+        break;
+
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
+
+      default:
```
Comment (VETALEK) [inline src/transformState.js:28]: better throw some error here

**Review by VETALEK — CHANGES_REQUESTED** [review]
Good start 🚀

**Review by VETALEK — APPROVED** [review]
👍

**Pull request closed without merging**